packages/web/src/scss/components/Modal/README.md

```diff
@@ -0,0 +1,54 @@
+# Modal
+
+Modal acts as dialog for any content.
+
+```html
+<button
```

Comment:
I am missing the backdrop in the example. Where should I place it?

Reply:
It's the same as in Header. So backdrop is added in JS.